modules/brainbrowser/ajax/minc.php

```diff
@@ -22,13 +22,59 @@
 
 $headers = array();
 
-$query     = "select File from files where FileID = :MincID";
-$minc_file = $DB->pselectOne($query, array('MincID' => $_REQUEST['minc_id']));
-$minc_file = getMincLocation() . $minc_file;
+if (strpos($_REQUEST['minc_id'], 'l') !== false) {
```

Comment:
Oh I'm sorry about that. I thought strpos returned a `-1` on failure, not `false`. Bizarre behaviour...